### DISCUSSION TITLE
Add a minimum items threshold for ```relationship``` field type

### DISCUSSION THREAD

**Discussion by susanu** [Laravel-Backpack/community-forum#1125] [Ideas]
Hi guys,

I have some relationships that have a small number of items.
Since the list is small, i don't want to use ```select2``` field type, i need just a normal dropdown.

What do you think?

**Comment by karandatwani92** [comment]
Hey @pxpm @tabacitu 

The user wants to display a basic HTML [multi-select](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select_multiple) instead of `select2` if the relationship field has limited entries. His preference is to have an expanded select box for limited entries.

He proposes using a threshold `attribute` to switch the UI according to the entry count.

How do you find this feature request?

**Comment by tabacitu** [comment]
Hey @susanu ,

In those cases I just use the [select_from_array](https://backpackforlaravel.com/docs/6.x/crud-fields#select_from_array) field. It's quick, it's simple, has a great UX and you can put whatever you want in it. Highly recommend it!

Cheers!
